christfellowshipchurch/web-app-v2
# Quick group finder edit

### About
Small change to the Group Finder Classes Hub to delete the word "Classes" from the button that previously said "Search All Classes Groups & Classes"

### Test Instructions
Open the Group Finder
Open the Classes Hub 
The button should say "Search All Groups & Classes" and not, "Search All Classes Groups & Classes"

### Screenshots
<img width="747" alt="Screen Shot 2021-08-30 at 10 32 46 AM" src="https://user-images.githubusercontent.com/46769629/131355885-f6ad2295-942b-4a48-a6d6-16e2047f9781.png">

### Closes Tickets
[CFDP-1673]

### Related Tickets
N/A
